source/common/formatter/substitution_formatter.cc

```diff
@@ -38,7 +38,7 @@ const std::regex& SubstitutionFormatParser::commandWithArgsRegex() {
   //                              _________________  _______________  _____________
   //                              |               |  |             |  |           |
   CONSTRUCT_ON_FIRST_USE(std::regex,
-                         R"EOF(^%((?:[A-Z]|[0-9]|_)+)(?:\(([^\)]*)\))?(?::([0-9]+))?%)EOF");
+                         R"EOF(^%((?:[A-Z]|[0-9]|_)+)(?:\(([^\)]*|[^\}]*)\))?(?::([0-9]+))?%)EOF");
```

Comment:
Can you update the comment that spells out the meaning of the regex above and below the regex?

Reply:
update base on https://regex101.com/r/rmBzTD/1